lib/KTable/index.vue

```diff
@@ -207,6 +227,12 @@
         return style;
       };
 
+      onMounted(() => {
+        nextTick(() => {
+          debounce(checkScrollable, 300)();
```

Comment:
Since `checkScrollable` is only called during the `onMounted` hook, if the table isn’t initially scrollable and is later resized to become scrollable, `isTableScrollable` doesn't gets set to true and the horizontal lines don’t appear. 

I wonder if, rather than setting the horizontal lines and table stickiness solely based on whether the table is scrollable, we should consider automatically applying the table stickiness and displaying the lines regardless of the current scrollability. Or, if it would be better to just watch the `elementWidth` and call `checkScrollable` whenever it changes.